Do not use a file called VERSION. It interferes with the C++20 <version>
header on case-insensitive filesystems.
https://github.com/aircrack-ng/aircrack-ng/commit/35169a66becf48fd014cb5124da3b61b4d25d812
--- Makefile.am.orig
+++ Makefile.am
@@ -65,7 +65,7 @@ bin_PROGRAMS =
 sbin_PROGRAMS =
 noinst_LTLIBRARIES =
 lib_LTLIBRARIES =
-EXTRA_DIST = VERSION
+EXTRA_DIST = AC_VERSION
 check_PROGRAMS =
 TESTS =
 INTEGRATION_TESTS =
--- configure.ac.orig
+++ configure.ac
@@ -37,6 +37,7 @@
 # If you delete this exception statement from all source files in the
 # program, then also delete it here.
 
+m4_pattern_allow([AC_VERSION])
 m4_define([version_date], "January 2020")
 m4_define([version_major], 1)
 m4_define([version_minor], 6)
@@ -230,7 +231,7 @@ LIBS="$opt_libs $LIBS"
 #
 AC_CONFIG_FILES([
 Makefile
-VERSION
+AC_VERSION
 
 include/aircrack-ng/version.h
 
--- evalrev.orig
+++ evalrev
@@ -49,7 +49,7 @@ elif test -d "${DIR}/.svn"; then
         SCM="_r1"
     fi
 else
-    REVISION="$(cat ${DIR}/VERSION)"
+    REVISION="$(cat ${DIR}/AC_VERSION)"
 fi
 
 if test "x$CMD" = "xscm"; then
